docs/analyzer.md

```diff
@@ -0,0 +1,24 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library
+that cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
The purist in me would remove them, but it seems they have no end effect either in the docs or in GitHub's rendering.